src/lib/containers/row_renderers/AMPAD/Consortium.tsx

```diff
@@ -54,6 +55,9 @@ export default class Consortium extends React.Component<ConsortiumProps, Consort
         const name = data[schema["Full Name"]]
         const org = data[schema.Consortium]
 
+        const values: string[][] = [["GRANT",""]]
```

Comment:
Confused by this change.  Is Grant going to be filled in (eventually)?

Reply:
Yes, this is a placeholder for now, will have to check with ljubo and others on where this will come from.